account_statement_import_txt_xlsx/wizards/account_statement_import_sheet_mapping_wizard.xml

```diff
@@ -1,173 +0,0 @@
-<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
No wizard now for selecting the mapping?

Reply:
Now it is not necessary, together with the module `account_statement_import_file` the wizard "`account.statement.import`" is implemented and the mapping is selected there.

Reply:
Already added.
![ejemplo-1](https://github.com/OCA/bank-statement-import/assets/4117568/5f4dedd4-7e86-46ee-b4e4-14ef557e607e)